test/e2e/scale.go

```diff
@@ -119,6 +119,15 @@ type scaleSpecInput struct {
 	// If not specified, this is a no-op.
 	PostNamespaceCreated func(managementClusterProxy framework.ClusterProxy, workloadClusterNamespace string)
 
+	// Allows to inject a function to be run after test workload cluster name and namespace are generated and
+	// before applying the clusterclass and the cluster template.
+	// If not specified, this is a no-op.
+	PostScaleClusterNamespaceCreated func(
+		clusterProxy framework.ClusterProxy,
+		clusterNamespace string,
+		clusterName string,
+		clusterClassYAML []byte,
+		clusterTemplateYAML []byte) (clusterClassTemlate []byte, ClusterTemplate []byte)
```

Comment:
```suggestion
		clusterTemplateYAML []byte) (clusterClassYAML []byte, clusterTemplateYAML []byte)
```
nit


Let's also add a new line after this line

Reply:
It is complaining that clusterClassYAML redeclared in this bock !

Reply:
I removed that return var name

Reply:
Ah :)